src/components/ebay-toggle-button/toggle-button.stories.js

```diff
@@ -0,0 +1,162 @@
+import { tagToString } from "../../../.storybook/storybook-code-source";
+import { addRenderBodies } from "../../../.storybook/utils";
+import readme from "./README.md";
+import component from "./index.marko";
+import WithIconTemplate from "./examples/with-icon.marko";
+import WithIconCode from "./examples/with-icon.marko?raw";
+import WithImageTemplate from "./examples/with-image.marko";
+import WithImageCode from "./examples/with-image.marko?raw";
+import MultilineSubtitleTemplate from "./examples/multiline-subtitle.marko";
+import MultilineSubtitleCode from "./examples/multiline-subtitle.marko?raw";
+
+const Template = (args) => ({
+    input: addRenderBodies(args),
+});
+
+export default {
+    title: "buttons/ebay-toggle-button",
+    component,
+    parameters: {
+        docs: {
+            description: {
+                component: readme,
+            },
+        },
+    },
+    argTypes: {
+        renderBody: {},
+        layoutType: {
+            type: "string",
+            control: { type: "select" },
+            options: ["minimal", "list", "gallery"],
```

Comment:
Since this combination isn't supported currently, maybe we should just remove the "minimal" option from here. We already cover it in the "Default" story anyway. The default/minimal option can only have a title. 
<img width="952" alt="image" src="https://github.com/eBay/ebayui-core/assets/1675667/04860042-2784-4937-b12c-ce9e40bc231f">

Reply:
FYI - "minimal" is implied, it's the base and doesn't even have a modifier.